akka-actor-typed-tests/src/test/java/jdocs/akka/typed/RouterTest.java

```diff
@@ -54,6 +63,50 @@ public DoLog(String text) {
     }
   }
 
+  static class Proxy {
+
+    public final ServiceKey<Command> registeringKey =
+        ServiceKey.create(Command.class, "aggregator-key");
+
+    public String mapping(Command command) {
+      return command.getId();
+    }
+
+    interface Command {
+      public String getId();
+    }
+
+    static class Message implements Command {
```

Comment:
we could make the example shorter by not having the interface `Command` and only using `Message` as the single protocol type.

Reply:
done